docs/vickywane-ambianic-stripe-swagger-docs-0.1-resolved.yaml

```diff
@@ -0,0 +1,51 @@
+---
+openapi: 3.0.1
+info:
+  title: Ambianic Serverless functions API Definition
+  description: An API Definition for Ambianic Serverless functions created using Swagger
+  version: "0.1"
+servers:
+- url: https://ambianic-functions.netlify.app
+- url: https://14fecb7f-38f9-43ce-ae90-9381932906cf.mock.pstmn.io
+paths:
+  /create-stripe-subscriber:
```

Comment:
@vickywane isn't the best practice for REST API's to describe the name of the resource and let the HTTP methods implement CRUD semantics (CREATE/POST-or-GET, READ/GET, UPDATE/POST, DELETE/DELETE)?

Should this URL resource name be `/subscriber` instead of `/create-stripe-subscriber` ?
Then use `operationId` as needed to define developer friendly JavaScript function names.

See these docs:
- https://nordicapis.com/10-best-practices-for-naming-api-endpoints/
- https://tyk.io/10-essentials-when-creating-an-openapi-specification/
- https://swagger.io/specification/#paths-object
- https://swagger.io/specification/#operation-object

Reply:
Thank you for sharing. I have read and applied [best practices for naming api endpoints](https://nordicapis.com/10-best-practices-for-naming-api-endpoints/)